.pylintrc

```diff
@@ -0,0 +1,5 @@
+[MASTER]
+disable=
+        C0114,
```

Comment:
Why do you disable this?

Reply:
Because I don't want to make docstrings mandatory in any case.
They get outdated pretty fast, and makes the developer responsable of keeping the docs up to date instead of having self documented code.